Content.Server/Chat/Commands/LOOCCommand.cs

```diff
@@ -35,7 +35,7 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
             if (string.IsNullOrEmpty(message))
                 return;
 
-            _e.System<ChatSystem>().TrySendInGameOOCMessage(entity, message, InGameOOCChatType.Looc, false, shell, player);
+            _e.System<ChatSystem>().TrySendInGameOOCMessage(entity, message, player);
```

Comment:
wierd....  u have communication way (channel) for ooc but there u are using separate method... is that correct?

Reply:
It's just a helper method; could very easily be replaced with SendChannelMessage entirely (as soon as I handle the whole magic strings issue)

Reply:
oh sorry then! if its just helper for usual way - its fine! i was just confused coz its kinda looking like old dirty methods.